### ISSUE TITLE
Not really an Issue, just wanted to ask you a question.

### ISSUE THREAD

**Issue by Classified787** [JoSchaap/GoT_A3Wasteland.Utes#2]
I am currently trying to make Taviana work with ArmA 3 for Wasteland. I have tried several test runs on my test server and have tried several different methods trying to get this to work but I have yet to even get to the point where the server will run the mission file. I have never tried to make a new version before, I have searched most of the forums and tried most of what I found in them but still no luck.
Since you have gotten Utes and Bornholm to work I was hoping you could give me some insight on the steps needed to make an old map work on with the new version of Wasteland.
I appreciate any help or advice you can give.
Thanks,
Ken

**Comment by JoSchaap** [comment]
i've shown/explained an example of the bornholm port here:

http://forums.a3wasteland.com/index.php?topic=1150.msg7309#msg7309

most work goes into relocating ALL markers to proper spots and pathing routes for the convoy/coastal missions

**Comment by Classified787** [comment]
Thank you very much. Some how I missed this in my searches. I did use the map editor to relocate all the markers and spawn locations and convoy routes. I am going to start over and go step by step this time. Thanks again. I will read through the post and see if I can get Tavi to work. 
Thanks,
Ken Bruner

```
 On Tuesday, March 24, 2015 5:45 AM, Johan Schaap <notifications@github.com> wrote:
```

 i've shown/explained an example of the bornholm port here:http://forums.a3wasteland.com/index.php?topic=1150.msg7309#msg7309most work goes into relocating ALL markers to proper spots and pathing routes for the convoy/coastal missions—

**Comment by Classified787** [comment]
I have gone through and relocated all the markers, stores, spawn locations, mission markers, I disabled the missions from running for now. Just so I could see if I could get Tavi to run without missions so I could check out spawn locations and the stores positions, I did relocate some ATM's ( didn't add to many just for testing). In the map editor I can get it to run no problems. I put it on the dedicated Test server and the server either crashes or just never shows up when I try to connect to it by IP address. I can post the mission.sqm I made for Tavi and all the other changes I have made if you would like to take a look to see if I am missing something obvious, since this is my first time at trying to port a map over to ArmA 3 wasteland. Oh, I did install the Taviana map on the dedicated server and added All in ArmA terrain and the Tavi mod to the command line.

**Comment by JoSchaap** [comment]
correct order @AllinArma;@Taviana

is your mission on git?

**Comment by Classified787** [comment]
No, I haven't posted it on Git yet. I can get it up on there under a different branch off of my regular Altis wasteland git post.I will let you know as soon as I get it posted, I am currently at work so I it might take a little bit for me to get it on there. Sort of have to bypass the internet security while I am at work. 
Thanks,
Ken Bruner

```
 On Tuesday, March 31, 2015 8:37 AM, Johan Schaap <notifications@github.com> wrote:
```

 correct order @AllinArma;@Tavianais your mission on git?—

**Comment by JoSchaap** [comment]
No rush..

You do realize this git (and discussion) is publicly visible including your work's email footer? :)

**Comment by Classified787** [comment]
Yes, Sometimes I forget to disable that part of the email footer. lol Even though those guys aren't to strict about this stuff I guess I might want to get rid of that for this type of "work". ;-)Thanks for the reminder.  
Thanks,
Ken Bruner

```
 On Tuesday, March 31, 2015 10:23 AM, Johan Schaap <notifications@github.com> wrote:
```

 No rush..You do realize this git (and discussion) is publicly visible including your work's email footer? :)—

**Comment by Classified787** [comment]
Okay, just posted the mission.sqm to github under it's own new branch. This has town, mission markers, some vehicle store, gen stores, and gun store locations. They are set outside right now until I edit in some building for them to be inside of and I just added a few ATM locations just for testing. Below is the link.
Classified787/Fail-Army-Wasteland-Taviana

|   |
|   |  |   |   |   |   |   |
| Classified787/Fail-Army-Wasteland-TavianaContribute to Fail-Army-Wasteland-Taviana development by creating an account on GitHub. |
|  |
| View on github.com | Preview by Yahoo |
|  |
|   |

   
Thanks,
Ken (Classified787) 

```
 On Tuesday, March 31, 2015 10:23 AM, Johan Schaap <notifications@github.com> wrote:
```

 No rush..You do realize this git (and discussion) is publicly visible including your work's email footer? :)—
